docs/http/migration-guide.md

```diff
@@ -0,0 +1,212 @@
+# HTTP semantic convention stability migration guide
+
+Due to the significant number of modifications and the extensive user base
+affected by them, existing HTTP instrumentations published by
+OpenTelemetry are required to implement a migration plan that will assist users in
```

Comment:
Might be good to spell out that this period is intended to be 6 months, or some timeline here. That's only inferable when you look later on and see the need to maintain the old values for that period of time.

Reply:
can you clarify? do you mean something other than

> - Need to maintain (security patching at a minimum) their existing major version
  for at least six months after it starts emitting both sets of conventions.